### PR TITLE
add changelog entry for #1646

### PR DESCRIPTION
**Pull request by zaki50**

### CODE REVIEW

**Comment by zaki50** [comment]
just adding changelog entry.

please review this @realm/java

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,6 @@
+0.84.1
+ * Fixed a bug simultaneous opening and closing Realm may produce inconsistent Realm instance and it gets NPE on column access. See #1646 for details.
```
Comment (cmelchior) [inline changelog.txt]: I would rewrite it to this:

`Fixed a bug where simultaneous opening and closing a Realm on different threads might result in a NullPointerException (#1646).`

Comment (zaki50) [inline changelog.txt]: thanks!

**Comment by cmelchior** [comment]
:+1:

**Comment by beeender** [comment]
:+1:

**Comment by kneth** [comment]
:+1:

**Pull request merged**